services/lock-keeper/lockkeeper.py

```diff
@@ -0,0 +1,81 @@
+import os
+
+import paramiko
+from flask import Flask
+from flask import jsonify
+from flask import request
+
+app = Flask(__name__)
+
+HOST = os.environ.get("LOCK_KEEPER_HOST")
+USER = os.environ.get("LOCK_KEEPER_USER")
+SECRET = os.environ.get("LOCK_KEEPER_SECRET")
+PORT = 22
+
+
+def execute_remote_command(command):
+    client = paramiko.SSHClient()
+    client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    client.connect(hostname=HOST, username=USER, password=SECRET, port=PORT)
+    stdin, stdout, stderr = client.exec_command(command)
+    data = stdout.read() + stderr.read()
+    client.close()
+    return data.decode('utf-8')
+
+
+def test_answer():
+    test_data = ('NXST_FLOW reply (xid=0x4):\n cookie=0x15, duration=422883.97'
+                 '5s, table=0, n_packets=70278, n_bytes=17499222, idle_age=5, '
+                 'hard_age=65534, in_port=7 actions=output:8\n cookie=0x16, du'
+                 'ration=422884.022s, table=0, n_packets=69701, n_bytes=173555'
+                 '49, idle_age=4, hard_age=65534, in_port=51 actions=output:52'
+                 '\n')
+
+    assert parse_dump_flows(test_data) == [
+        {
+            'in_port': 7,
+            'out_port': 8
+        }, {
+            'in_port': 51,
+            'out_port': 52
+        }
+    ]
+
+
+def int_from_str_by_pattern(string, pattern):
+    start = string.find(pattern)
+    end = string.find(' ', start)
+    if end == -1:
+        end = len(string)
```

Comment:
`end = None` whould be enough here.

Reply:
> Return -1 if sub is not found. 

from  https://docs.python.org/3/library/stdtypes.html#str.find